### PR TITLE
Data schema

### PR DESCRIPTION
**Pull request by kirti3192**

### CODE REVIEW

File: static/website/templates/get-question.html [hunk 1]
```diff
@@ -26,7 +26,6 @@ <h5>Title</h5>
             <input class="form-control" type="text">
             <h5>Question</h5>
         </div>
-        <a class="delete-question btn btn-xs btn-info" data-qid="{{ question.id }}" href="#">Delete</a>
```
Comment (theskumar) [inline static/website/templates/get-question.html]: does this need to be removed, doesn't look like part of this pull-request. @kirti3192

**Comment by theskumar** [comment]
Please update readme with the detail about this schema as well.

File: static/website/templates/get-question.html (hunk 1 above)
Comment (kirti3192) [reply static/website/templates/get-question.html]: This button was made there to check whether we can delete the cross user questions or not. 
That was just for testing, so have to remove it.

Comment (kirti3192) [reply static/website/templates/get-question.html]: @theskumar : can we merge this?

**Pull request merged**